### PR TITLE
Mostly aesthetic mods and a small bugfix

### PR DESCRIPTION
**Pull request by laskasn**
This PR includes the following modifications:

- Rename the `config` variable into `keycloak_config`
- Changes the tags to have a `keycloak:` prefix
- Beautify the template placeholders, adds a space between curly brackets and variable
- Fixed a bug in the `get_realm_keys.yml` file
- Renamed tags `keycloak:configure:*` to `keycloak:config:*` ,  `keycloak:install_plugins` to `keycloak:install:plugins` and `keycloak:install_standaloneha` to `keycloak:install:standaloneha`

### CODE REVIEW

File: roles/keycloak/tasks/configure.yml [hunk 1]
```diff
@@ -6,23 +6,23 @@
...
       timeout: 600
 
 
 - name: Acquire tokens
   include_tasks: blocks/helpers/get_tokens.yml
   tags:
-    - "realm_configuration"
-    - "wayf_plugin"
+    - "keycloak:configure:realm_configuration"
+    - "keycloak:configure:wayf_plugin"
```
Comment (NicolasLiampotis) [inline roles/keycloak/tasks/configure.yml:25]: configure --> config

File: roles/keycloak/tasks/configure.yml [hunk 2]
```diff
@@ -98,43 +98,43 @@
...
       include_tasks: blocks/configure_idp_mappers.yml
       with_subelements:
-        - "{{config.realms | default([])}} "
+        - "{{ keycloak_config.realms | default([]) }} "
         - "idp_mappers"
         - skip_missing: True
       run_once: true
 
-  tags: "realm_configuration"
+  tags: "keycloak:configure:realm_configuration"
```
Comment (NicolasLiampotis) [inline roles/keycloak/tasks/configure.yml:137]: configure --> config

File: roles/keycloak/tasks/configure.yml [hunk 3]
```diff
@@ -144,19 +144,19 @@
...
-      with_items: "{{keycloak_plugins.wayf.theme.add_to_realms}}"
+      with_items: "{{ keycloak_plugins.wayf.theme.add_to_realms }}"
 
   when: wayf_enabled == true
   vars:
-    wayf_enabled: "{{keycloak_plugins.wayf.enabled | default(false)}}"
-  tags: "wayf_plugin"
\ No newline at end of file
+    wayf_enabled: "{{ keycloak_plugins.wayf.enabled | default(false) }}"
+  tags: "keycloak:configure:wayf_plugin"
```
Comment (NicolasLiampotis) [inline roles/keycloak/tasks/configure.yml:162]: configure -- config

File: roles/keycloak/tasks/install_plugin.yml [hunk 4]
```diff
@@ -59,4 +59,4 @@
 
   when: not item is undefined
   become: yes
-  tags: "install_plugins"
\ No newline at end of file
+  tags: "keycloak:install_plugins"
```
Comment (NicolasLiampotis) [inline roles/keycloak/tasks/install_plugin.yml:62]: keycloak:install_plugins --> keycloak:install:plugins

File: roles/keycloak/tasks/main.yml [hunk 5]
```diff
@@ -192,7 +192,7 @@
   include_tasks: "install_plugin.yml"
   when: not keycloak_plugins is undefined
   loop: "{{ lookup('dict', keycloak_plugins, wantlist=True) }}"
-  tags: "install_plugins"
+  tags: "keycloak:install_plugins"
```
Comment (NicolasLiampotis) [inline roles/keycloak/tasks/main.yml:195]: keycloak:install_plugins --> keycloak:instal:plugins

File: roles/keycloak/tasks/main.yml [hunk 6]
```diff
@@ -221,4 +221,4 @@
 
 - name: Configure various keycloak aspects
   import_tasks: "configure.yml"
-  tags: "configure"
+  tags: "keycloak:configure"
```
Comment (NicolasLiampotis) [inline roles/keycloak/tasks/main.yml:224]: keycloak:configure -->keycloak:config

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]

**Comment by laskasn** [comment]
Performed the changes

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**